proposals/incubation/istio.md

```diff
@@ -0,0 +1,375 @@
+
+### Statement on alignment with CNCF mission
+
+Istio aims to provide security, observability and policy for traffic on top of Kubernetes. If the goal of the CNCF is to make cloud native computing ubiquitous, then a goal of Istio could be expressed as making that ubiquity safe and easy to manage.
+
+Istio is a service mesh, and service meshes are called out in the CNCF charter as exemplifying the Cloud Native approach.
+
+Istio stands on the shoulders of several CNCF projects, such as Kubernetes, Envoy, gRPC, Prometheus and SPIFFE. It can be installed with Helm, and integrations exist for projects including Knative, Flagger, Jaeger, Open Policy Agent, and OpenTelemetry.
+
+### Communication channels
+
+* Slack: https://slack.istio.io
+* Discourse: https://discuss.istio.io
+* Shared Drive: https://drive.google.com/corp/drive/folders/0ADmbrU7ueGOUUk9PVA
+
+### Website
+
+https://istio.io/
+
+### Release methodology and mechanics
+
+We produce new builds of Istio for each commit. Around once per quarter, we build a minor release and run through several additional tests as well as release qualification. We release patch versions for issues found in minor releases.
+
+Detailed information on Istio's release management can be found in [its documentation](https://istio.io/latest/docs/releases/supported-releases/) and [its developer wiki](https://github.com/istio/istio/wiki/Istio-Release-Management).
+
+### Security processes
+
+Istio has a responsible and mature vulnerability submission and disclosure program, managed by the Product Security Working Group.  The processes are documented at https://istio.io/latest/docs/releases/security-vulnerabilities/.
+
+Vendors meeting certain published criteria receive advance notice of security vulnerabilities through Istio's early disclosure list, which is documented at https://github.com/istio/community/blob/master/EARLY-DISCLOSURE.md.
+
+### Roadmap
+
+[Our 2022 roadmap](https://docs.google.com/document/d/1gfruq22BXKYgKA-BttHv2gE-woq6lu1kreA8VyIdQXc/edit#heading=h.gjdgxs) highlights the following goals:
+
+* Stabilizing Istio's API surface
+* Improving upgrades and troubleshooting
+* Enhancing extensibility
+* Expanding reach
+* Further security hardening
+
+### Social media account
+
+* Twitter: https://twitter.com/istiomesh
+* YouTube: https://www.youtube.com/c/istio
+
+### Community size
+
+Istio has had over [8000 individual contributors](https://istio.teststats.cncf.io/d/18/overall-project-statistics-table?orgId=1) on the project’s GitHub repositories. 
+
+The project has over [32,000 GitHub stars](https://istio.teststats.cncf.io/d/18/overall-project-statistics-table?orgId=1) across its repositories.
+
+[Over 70 companies](https://istio.teststats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20decade&var-metric=prs) have submitted at least 8 Pull Requests to Istio.
+
+#### Contribution
+
+Istio's proud parents are Google and IBM, alongside the Lyft team who built the Envoy proxy.
```

Comment:
Cisco was a parent as well. The use of the word proud is odd.

Reply:
"proud parents" is a simply figure of speech in English.  Happy to remove it if it does not translate well.

Reply:
Right - thanks for the clarity. I would recommend avoiding idiomatic expressions.